### ISSUE TITLE
Don't suggest Laravel Pest tests to be static functions

### ISSUE THREAD

**Issue by tanthammar** [kalessil/phpinspectionsea#1812]
Description (screenshot):
![image](https://user-images.githubusercontent.com/21239634/153462847-55121c36-55c1-4dd3-9feb-86ff9002bf93.png)

Would be great if this plugin could be more compatible with Laravel Pest.
The function example cannot be static...

https://pestphp.com/docs/plugins/livewire

**Comment by AlexandreGerault** [comment]
Yes, indeed the Pest function cannot be static since they internally rely on an actual class instance. Maybe you can get rid of these warning with a `$this` used in the test but this look bad in my opinion to have `$this` on such tests (not really logic to see `$this` on a code we didn't explicitly tell it was a class).

Any news about the fact this would be considered or not? 😄
